v2/cli/term/page_router.go

```diff
@@ -49,46 +49,47 @@ func NewPageRouter(
 
 // loadProjectsPage refreshes the projects page and brings it into focus.
 func (r *pageRouter) loadProjectsPage() {
-	r.loadPage(projectsPageName, true, func() {
+	r.loadPage(projectsPageName, func() {
 		r.projectsPage.refresh()
 	})
 }
 
 // loadProjectPage refreshes the project page and brings it into focus.
 func (r *pageRouter) loadProjectPage(projectID string) {
-	r.loadPage(projectPageName, true, func() {
+	r.loadPage(projectPageName, func() {
 		r.projectPage.refresh(projectID)
 	})
 }
 
 // loadEventPage refreshes the event page and brings it into focus.
 func (r *pageRouter) loadEventPage(eventID string) {
-	r.loadPage(eventPageName, true, func() {
+	r.loadPage(eventPageName, func() {
 		r.eventPage.refresh(eventID)
 	})
 }
 
 // loadJobPage refreshes the job page and brings it into focus.
 func (r *pageRouter) loadJobPage(eventID, jobID string) {
-	r.loadPage(jobPageName, true, func() {
+	r.loadPage(jobPageName, func() {
 		r.jobPage.refresh(eventID, jobID)
 	})
 }
 
 // loadLogPage loads a floating window that displays logs and brings it into
 // focus.
 func (r *pageRouter) loadLogPage(page *page, eventID, jobID string) {
-	// r.loadPage(logPageName, func() {
-	// 	r.logPage.refresh(*page, eventID, jobID)
-	// })
-	r.loadPage(logPageName, false, func() {
+	r.loadPage(logPageName, func() {
 		r.logPage.refresh(*page, eventID, jobID)
-	})
+	}, r.logPage.logText)
```

Comment:
@krancour I modified `loadPage` to take in a page to focus on as a variadic input since there are no conditional inputs in Go. Would this make more sense than passing in a parameter strictly for a focus page, and having non-focus pages pass in a `nil` value?

Reply:
See https://github.com/willie-yao/brigade/pull/1 for a cleaner alternative.

Reply:
LGTM, thanks for the suggestion